### ISSUE TITLE
Whether to add asset export function to export table

### ISSUE THREAD

**Issue by qianshuiliyu** [louislam/uptime-kuma#4678]
### 📑 I have found these related issues/pull requests

暂时没有

### 🏷️ Feature Request Type

Other

### 🔖 Feature description

是否增加资产导出功能，导出表格

### ✔️ Solution

是否增加资产导出功能，导出表格

**Comment by CommanderStorm** [comment]
I have no clue what you are asking as the transaltors (deepl, Google translate) out there are not good enough to decrypt this text into English.
Could you rephrase your question in English?

I think you are talking about the export functionality. I don't know what "assets" are in this context.
This functionality has been deprecated due to numerous bugs and has been removed as of v2.0. See #4500 for the tracking issue.
I think this resolves this issue. If it does not, feel free to reach out.

**Comment by CommanderStorm** [comment]
> I think this resolves this issue. If it does not, feel free to reach out.

Forgot to close this one
